src/walletSdk/Horizon/AccountService.ts

```diff
@@ -13,7 +15,68 @@ export class AccountService {
     this.network = cfg.stellar.network;
   }
 
+  /**
+   * Generate new account keypair (public and secret key). This key pair can be
+   * used to create a Stellar account.
+   *
+   * @return public key and secret key
+   */
   createKeypair(): SigningKeypair {
     return new SigningKeypair(Keypair.random());
   }
+
+  /**
+   * Get account information from the Stellar network.
+   *
+   * @param accountAddress Stellar address of the account
+   * @param serverInstance optional Horizon server instance when default doesn't work
+   * @return account information
+   */
+  async getInfo({ 
+    accountAddress, 
+    serverInstance = this.server 
+  }: { 
+    accountAddress: string; 
+    serverInstance?: Server;
+  }): Promise<ServerApi.AccountRecord> {
+    return serverInstance.accounts().accountId(accountAddress).call();
+  }
+
+  /**
+   * Get account operations for the specified Stellar address.
+   *
+   * @param accountAddress Stellar address of the account
+   * @param limit optional how many operations to fetch, maximum is 200, default is 10
+   * @param order optional data order, ascending or descending, defaults to descending
+   * @param cursor optional cursor to specify a starting point
+   * @param includeFailed optional flag to include failed operations, defaults to false
+   * @return a list of operations
+   * @throws [OperationsLimitExceededException] when maximum limit of 200 is exceeded
+   */
+  async getHistory({
```

Comment:
I think in the future we should improve this method so it iterates over pages. (This will remove cursor and remove limit restriction)